### DISCUSSION TITLE
This is really useful!

### DISCUSSION THREAD

**Discussion by jordibruin** [LucasMucGH/BottomSheet#77] [Show and tell]
Hi Lucas, couldn't find out how to reach you but this is the best BottomSheet replacement I've found. Using it in my new app and it's great!

https://twitter.com/jordibruin/status/1433718836902178818?s=21

Thanks,
Jordi

**Comment by lucaszischka** [comment]
Yes, I’m not on twitter only instagram, but thank you a lot for your appreciation ❤️
I’m always happy to hear from people who like my work.
